docs/_posts/murat-gunay/2021-12-07-roberta_token_classifier_bne_capitel_ner_es.md

```diff
@@ -0,0 +1,132 @@
+
+- RoBERTa-base-bne is a transformer-based masked language model for the Spanish language. It is based on the RoBERTa base model and has been pretrained using the largest Spanish corpus known to date, with a total of 570GB of clean and deduplicated text processed for this work, compiled from the web crawlings performed by the National Library of Spain (Biblioteca Nacional de España) from 2009 to 2019.
+
+## Predicted Entities
+
+`OTH`, `PER`, `LOC`, `ORG`
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+<button class="button button-orange" disabled>Open in Colab</button>
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/public/models/roberta_token_classifier_bne_capitel_ner_es_3.3.2_2.4_1638866935540.zip){:.button.button-orange.button-orange-trans.arr.button-icon}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+documentAssembler = DocumentAssembler()\
+      .setInputCol("text")\
+      .setOutputCol("document")
+
+sentenceDetector = SentenceDetectorDLModel.pretrained("sentence_detector_dl", "xx")\
+       .setInputCols(["document"])\
+       .setOutputCol("sentence")
+
+tokenizer = Tokenizer()\
+      .setInputCols(["sentence"])\
+      .setOutputCol("token")
+
+tokenClassifier = RoBertaForTokenClassification.pretrained("roberta_token_classifier_bne_capitel_ner", "es"))\
+  .setInputCols(["sentence",'token'])\
+  .setOutputCol("ner")
+
+ner_converter = NerConverter()\
+      .setInputCols(["sentence", "token", "ner"])\
+      .setOutputCol("ner_chunk")
+      
+nlpPipeline = Pipeline(stages=[documentAssembler, sentenceDetector, tokenizer, tokenClassifier, ner_converter])
+
+empty_data = spark.createDataFrame([[""]]).toDF("text")
+
+model = nlpPipeline.fit(empty_data)
+text = """Mi nombre es Wolfgang y trabajo en la fábrica de Mercedes-Benz vivida en Madrid."""
+result = model.transform(spark.createDataFrame([[text]]).toDF("text"))
+```
+```scala
+val documentAssembler = DocumentAssembler()
+      .setInputCol("text")
+      .setOutputCol("document")
+
+val sentenceDetector = SentenceDetectorDLModel.pretrained("sentence_detector_dl", "xx")
+       .setInputCols(Array("document"))
+       .setOutputCol("sentence")
+
+val tokenizer = Tokenizer()
+      .setInputCols(Array("sentence"))
+      .setOutputCol("token")
+
+val tokenClassifier = RoBertaForTokenClassification.pretrained("roberta_token_classifier_bne_capitel_ner", "es"))\
+  .setInputCols(Array("sentence","token"))\
+  .setOutputCol("ner")
+
+ner_converter = NerConverter()\
+      .setInputCols(Array("sentence", "token", "ner"))\
+      .setOutputCol("ner_chunk")
+      
+val pipeline = new Pipeline().setStages(Array(documentAssembler, sentenceDetector, tokenizer, tokenClassifier, ner_converter))
+
+val example = Seq.empty["Mi nombre es Wolfgang y trabajo en la fábrica de Mercedes-Benz vivida en Madrid."].toDS.toDF("text")
```

Comment:
This sentence is not correct in Spanish. Take any other from https://es.wikipedia.org/